pyproject.toml

```diff
@@ -6,32 +6,30 @@ build-backend = "hatchling.build"
 name = "qadence2-platforms"
 description = "Platform-dependent engines and model to execute compiled expressions with common set of methods."
 readme = "README.md"
-version = "0.1.2"
-requires-python = ">=3.10"
+version = "0.2.0"
+requires-python = ">=3.10,<3.12"
 license = {text = "Proprietary"}
 keywords = ["quantum"]
 authors = [
   { name = "Eduardo Maschio", email = "eduardo.maschio@pasqal.com" },
   { name = "Kaonan Micadei", email = "kaonan.micadei@pasqal.com" },
-  { name = "Dominik Seitz", email = "dominik.seitz@pasqal.com" },
-  { name = "Pim Venderbosch", email= "pim.venderbosch@pasqal.com" },
+  { name = "Dominik Seitz", email = "dominik.seitz@pasqal.com" }
 ]
 classifiers = [
   "Development Status :: 4 - Beta",
   "Programming Language :: Python",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
-  "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 
 # always specify a version for each package
 # to maintain consistency
 dependencies = [
-  "qadence2-ir~=0.1",
-  "pulser~=1.1",
-  "pyqtorch~=1.4",
+  "qadence2-ir",
```

Comment:
Is renovate working ? Otherwise feel free to raise an issue and tag me in.

Reply:
I noticed that renovate only picks up dependencies that have a version specifier.